main_logic/tts_client.py

```
@@ -990,3 +1007,241 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
+                payload = {
+                    "header": {"action": "run-task", "task_id": str(uuid.uuid4())},
+                    "payload": {"input": {"text": text}}
+                }
+                try:
+                    if ws_conn is None:
+                        await ensure_connection()
+                    await ws_conn.send(json.dumps(payload))
+                    logger.info(f"发送合成片段: {text}")
+                except Exception as e:
+                    # ... (这里放你刚才写的重连逻辑) ...
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)